src/components/header/Header.js

```diff
@@ -0,0 +1,45 @@
+import menu from "./HeaderMenu";
+import logo from "../header/logo.svg"
+import React from "react";
+import Titles from "./titles";
+
+const Header = () => {
+    return (
+        <header>
+        <nav className="navMenu">
+            <div className="container">
+               <img className="logo" src={logo} alt='logoSpring' />
+               <ul className="menuItems">
+                   {menu.map((el) => {
+                       return <Titles menu={el}/>
+                   })}
+               </ul>
+            </div>
+</nav>
+<div className="smallVersion">
+   <div className="hamburger" onClick={hambClick}>
+       <div className="burger"></div>
+       <div className="burger"></div>
+       <div className="burger"></div>
+   </div>
+</div>
+</header>
+    );
+  }
+  
+ 
+
+ function hambClick(){
```

Comment:
This should be implemented as react code, without using dom api. 
See https://reactjs.org/docs/handling-events.html